### DISCUSSION TITLE
QFieldSync plugin: avoid copying layers that are not used in the project

### DISCUSSION THREAD

**Discussion by aloboa** [opengisch/QField#4926] [Q&A]
When the plugin gets to the step: "Directories to be copied.",
is it copying all files in those directories or just those files that are used in the current QGIS project?
Is there a way to customize this particular behavior?

**Comment by suricactus** [comment]
It copies all the files in the directory. Use layer action "Remove from project" to configure layers that you don't want on QField. See how it's done in the docs https://docs.qfield.org/get-started/tutorials/get-started-qfs/#layer-configuration .
